pkg/complianceoperator/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,13 +1,29 @@
 //go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
-// Code generated by operator-sdk. DO NOT EDIT.
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by deepcopy-gen. DO NOT EDIT.
```

Comment:
Is this PR complete? I don't understand how the remaining changes could explain this hunk?
And also I don't understand why the new apparently unrelated entries in go.mod appeared if we're only removing package dependencies?

Reply:
I don't know hot to generate this file? I tried `make generate` but it does not create it. So I used following command `deepcopy-gen  -i ./pkg/complianceoperator/api/v1alpha1 ` this probably should be fixed in a separated PR.